letsencrypt/crypto_util.py

```diff
@@ -101,13 +104,17 @@ def make_csr(key_str, domains):
 
     csr = M2Crypto.X509.Request()
     csr.set_pubkey(pubkey)
-    name = csr.get_subject()
-    name.C = "US"
-    name.ST = "Michigan"
-    name.L = "Ann Arbor"
-    name.O = "EFF"
-    name.OU = "University of Michigan"
-    name.CN = domains[0]
+    # TODO: "The CSR MUST contain at least one extensionRequest
```

Comment:
I allowed myself to remove these commented subject fields, and left an appropriate `TODO` instead.